cpp/tests/io/parquet_test.cpp

```diff
@@ -3016,7 +3021,7 @@ TEST_F(ParquetReaderTest, DecimalRead)
     cudf::test::expect_columns_equal(result.tbl->view().column(1), col1);
 
     read_opts.set_columns({"dec20p1"});
-    EXPECT_THROW(cudf_io::read_parquet(read_opts), cudf::logic_error);
+    EXPECT_NO_THROW(cudf_io::read_parquet(read_opts));
```

Comment:
If this doesn't throw then it should probably check if the results match

Reply:
This test uses an embedded file, so I don't have the reference output available. Any suggestions?

Reply:
1. Write out the embedded file with host_write
2. Read it using pandas and grab the third column
3. ...
4. Profit!?